presto-main/src/test/java/io/prestosql/sql/query/TestShowQueries.java

```diff
@@ -91,4 +91,23 @@ public void testListingEmptyCatalogs()
             assertions.getQueryRunner().getAccessControl().reset();
         });
     }
+
+    @Test
+    public void testShowColumnsLike()
+    {
+        assertions.assertQuery("SHOW COLUMNS FROM system.runtime.nodes LIKE 'node%'",
```

Comment:
can you add tests for SHOW COLUMNS:
 - without LIKE
 - with LIKE but with no wildcards
 - empty and null LIKE
 - empty, null and incorrect ESCAPE
 - with wrong types given for LIKE and ESCAPE 
- without LIKE and with ESCAPE (this one in TestSqlParser)